src/Native/SymSgdNative/SymSgdNative.h

```diff
@@ -142,4 +143,7 @@ struct SymSGDState
     int NumFrequentFeatures;
     int PassIteration;
     float WeightScaling;
-};
\ No newline at end of file
+};
+
+// Loggin call back signature for ML.NET
```

Comment:
>Loggin [](start = 3, length = 6)

Typo. Though, "Loggin' call" sounds like it could be the title to a catchy 19th century folk song.